ReactWindows/ReactNative.Shared.Tests/Modules/WebSocket/WebSocketModuleTests.cs

```diff
@@ -112,22 +124,26 @@ public void WebSocketModule_DataEvent()
                 }
             }));
 
+            int waitTimeout = System.Diagnostics.Debugger.IsAttached ? -1 : 5000;
+
```

Comment:
Use a constant for the timeouts.

---
In reply to: [87299617](https://github.com/ReactWindows/react-native-windows/pull/868#discussion_r87299617) [](ancestors = 87299617)